apps/mail/app/root.tsx

```
@@ -185,33 +198,33 @@ function NotFound() {
-        <div className="mt-2 flex justify-center gap-2">
-          <Button
-            variant="outline"
+        {/* 404 Message */}
+        <div className="mb-8 text-center space-y-2">
+          <h1 className="text-3xl font-medium text-white ">404: Page not found</h1>
+          <p className="text-white/60 text-sm">This page does not exist.</p>
+          <button
             onClick={() => navigate(-1)}
-            className="text-muted-foreground gap-2"
+            className="inline-flex items-center gap-2 px-4 py-2 rounded-lg border border-white/20 text-white text-sm hover:bg-white/5 transition-colors"
           >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)